RHEL6_7/storage/dasd-format/module.ini

```diff
@@ -0,0 +1,6 @@
+[preupgrade]
+content_title = Detection of Direct Access Storage Device (DASD) format on s390x platform for LDL format
+content_description = On the s390x architecture, if a partition is formatted with LDL on one or more Direct Access Storage Devices, the upgrade process will fail and cause a data loss on such partition.
+author = Arthur Mello <amello@redhat.com>
+requires = grep, s390utils-base, util-linux
+binary_req = blkid, grep
```

Comment:
It doesn't make sense to require grep rpm and grep binary.  Same for blkid. Require just rpm or binary. I prefer rpm requirement only, but you can choose any of them you prefer.

The problem is in `s390utils-base` requirement. In such case, you would get automatically HIGH risk and `action_needed` result on all non-s390x archs. But you would use `applies_to` for the RPM so module will be automatically applied only when the RPM will be installed (so you will not need the check of `DASDVIEW` inside the script on your own).